src/chart/generateCategoricalChart.tsx

```diff
@@ -2191,6 +2218,17 @@ export const generateCategoricalChart = ({
         );
       }
 
+      if (this.props.accessibilityLayer) {
+        attrs.tabIndex = 0;
+        attrs.role = 'img';
+        attrs.onKeyDown = (e: any) => {
+          this.accessibilityManager.keyboardEvent(e);
+        };
+        attrs.onFocus = () => {
+          this.accessibilityManager.focus();
+        };
```

Comment:
these are all props that will get overridden if the user specifies `accessibilityLayer` - ideally the user wouldn't really be able to specify most of them themselves (recharts extends SVGProps for everything), but we could have issues where users report that their `tabIndex`, `role`, `onKeyDown`, and `onFocus` are being overridden. We would have to call these out in the docs as a caveat to `accessibilityLayer`

Reply:
Good point. I guess this could also be done in the description of the accessibilityLayer in the API documentation.